writeups/Drupal_Drupal_TameeshB.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Drupal
+
+
+## What is Drupal 
+Drupal is content management software.  
```

Comment:
Drupal is **a** content management software.

Reply:
I'll edit it. No idea why they didn't put an "a" here https://www.drupal.org/about

Reply:
Oh, you should probably edit the about page too then (if that's open source as well). :wink:

Reply:
FIxed.